application-workloads/blockchain/blockchain/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
update this file to reflect that only groestlcoin is supported

Reply:
where do you want this info? In the summary or description? Maybe you can add a suggestion

Reply:
@bmoore-msft ^

Reply:
Done